velox/experimental/cudf/exec/Utilities.h

```diff
@@ -56,4 +57,25 @@ createMemoryResource(std::string_view mode, int percent);
     const TypePtr& tableType,
     rmm::cuda_stream_view stream);
 
+[[nodiscard]] std::vector<std::unique_ptr<cudf::table>>
+getConcatenatedTableBatched(
+    std::vector<CudfVectorPtr>& tables,
+    const TypePtr& tableType,
+    rmm::cuda_stream_view stream);
```

Comment:
nit: some documentation on this (mention stream joining, or stream safe) would be helpful.
and the size_type limit.